test/remoteCollection/collect.js

```diff
@@ -574,7 +573,7 @@ describe('test/remoteCollection/collect.js >', () => {
         expect(collectRes.res).to.not.equal(undefined);
         expect(collectRes.res.status).to
           .equal(httpStatus.SERVICE_UNAVAILABLE);
-        expect(collectRes.res.body.error.message)
+        expect(collectRes.res.response.body.error.message)
```

Comment:
Why this change?

Reply:
I had to change this test when I added the "else if" on line 139 of collect.js